koch.nim

```diff
@@ -26,6 +26,7 @@ import
 import tools / kochdocs
 
 const VersionAsString = system.NimVersion
+const env_NIM_COMPILE_TO_CPP = "NIM_COMPILE_TO_CPP"
```

Comment:
it is already merged, but is this constant really necessary? Please remove it again.

Reply:
done as a part of https://github.com/nim-lang/Nim/pull/10242 (yes, looks silly now after code was changed and it's only used once)